src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Helper/QueryHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Community.PowerToys.Run.Plugin.ValueGenerator.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with queries
+    /// </summary>
+    internal static class QueryHelper
+    {
+        /// <summary>
+        /// A list that contain all of the value generators and its descriptions
+        /// </summary>
+        internal static readonly IReadOnlyDictionary<string, string> GeneratorList = new Dictionary<string, string>(18)
+        {
+            { "uuid", "Generate a random UUID - Example: uuid or guid" },
```

Comment:
1. We need to translate this. (Maybe we have to add the whole description to the translated string. The translations is required for the following parts:
   - `Generate a random UUID - Example:`
   - `or` in the list of namespace types.
   - `your input`
2. I worry that the description gets to long with text zoom. Can we use our own object class with a property used to set the query text. Then we can use one of the following layouts:
   - Title: Name - Description;  Subtitle: Example
   - Title: Example;  Subtitle: Name - Description
3. I loved the first version of the description for uuid that tells what type is generated (time based or namespace base). Can we get this in in a short text form?

Reply:
1. I don't have any idea about how translation works, can you give hints about how to implement it? (documentation links, code example that's already implemented here, etc.)
2. I'm trying to apply this approach but all of the title is always copied to query, it only change to the intended value (keyword only) after I pressed enter or click the item
![image](https://github.com/microsoft/PowerToys/assets/24465401/7317bf05-1820-4114-948c-a39c6e568cda)
3. Got it.

Reply:
For 2:
In the `QueryHelper.cs` add a using for `using Community.PowerToys.Run.Plugin.ValueGenerator.Properties;`, then you can add the strings to `<PluginFolder>\Properties\Resources.resx`  and access it via `Resources.<Stringname>` in your code.
It might be good to use {0}, {1}, ... in the resx as placeholders for example parts and then replace them in your code using `string.Fromat()`. This will prevent us from breaking gramar by splitting the text.

For 3:
See my comment here: https://github.com/microsoft/PowerToys/pull/33490/files#r1657194550